dependencyManagement/build.gradle.kts

```diff
@@ -96,7 +96,9 @@ val DEPENDENCIES = listOf(
   "commons-logging:commons-logging:1.3.0",
   "commons-validator:commons-validator:1.8.0",
   "io.netty:netty:3.10.6.Final",
+  "io.opentelemetry.contrib:opentelemetry-aws-resources:1.33.0-alpha",
   "io.opentelemetry.contrib:opentelemetry-aws-xray-propagator:1.28.0-alpha",
+  "io.opentelemetry.contrib:opentelemetry-gcp-resources:1.33.0-alpha",
```

Comment:
Could you extract `otelContribVersion` into a variable like we already have for `otelSdkVersion`. This should let renovate update all of these with one PR.